src/main/java/mil/dds/anet/ws/AccessTokenPrincipal.java

```diff
@@ -0,0 +1,18 @@
+package mil.dds.anet.ws;
+
+import java.security.Principal;
+import javax.security.auth.Subject;
+import mil.dds.anet.beans.AccessToken;
+
+public record AccessTokenPrincipal(AccessToken accessToken) implements Principal {
+
+  @Override
+  public String getName() {
+    return "AccessTokenPrincipal";
```

Comment:
Shouldn't this `return accessToken.getName();` instead of the same fixed name for _every_ principal?